programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -109,7 +110,7 @@ static int run_tunnel(const char *ip_range, dns_manager *dns) {
 #if __APPLE__ && __MACH__
     tun = utun_open(tun_error, sizeof(tun_error), ip_range);
 #elif __linux__
-    tun = tun_open(tun_error, sizeof(tun_error), ip_range);
+    tun = tun_open(tun_ip, dns->dns_ip, ip_range, tun_error, sizeof(tun_error));
```

Comment:
Is a null check needed for `dns` here?

Reply:
we always create dns manager (internal or dnsmasq-based)